### ISSUE TITLE
[INFRA] The Parcel TS project is not udpated correctly by the script updating the bpmn-visualization version

### ISSUE THREAD

**Issue by csouchet** [process-analytics/bpmn-visualization-examples#351]
When I released the **0.24.1** version of the `bpmn-visualization` library, the `projects/typescript-vanilla-with-parcel/package.json` file has an unwanted update:
```
"alias": {
  "bpmn-visualization": "bpmn-visualization/dist/bpmn-visualization.esm.js"
}
```
replaced by
```
"alias": {
  "bpmn-visualization": "0.24.1"
}
```

See #350 for more details

To fix that, we may update the regexp in the script `scripts/update-lib-version.bash`.

**Comment by tbouffard** [comment]
We could try to replace the alias configuration in the package.json by a settings in a Parcel configuration file, as we do for Webpack.
See https://github.com/process-analytics/bpmn-visualization-examples/pull/338#issuecomment-1144687626

**Comment by tbouffard** [comment]
The alias is going to be removed in #384. It was needed as the "browser" field in the bpmn-visualization package.json was wrong.
This field has been fixed in v0.26.2 so the alias can be removed.
